lbz/consts.py

```diff
@@ -0,0 +1 @@
+DIRECT_LAMBDA_REQUEST = "direct_lambda_request"
```

Comment:
But also, shouldn't it be declared directly inside the `LambdaSources` enum class? 🤔 

Keeping it separately makes this code inconsistent.